config/codeceptjs_helpers.js

```diff
@@ -1,6 +1,6 @@
 const assert = require('assert')
 // eslint-disable-next-line camelcase
-let Helper = codecept_helper
+const Helper = codecept_helper
```

Comment:
Switched to `const` to follow Codecept's guide. https://codecept.io/helpers/#development